### PR TITLE
Change logging

### PR DESCRIPTION
**Pull request by shintasmith**
Change a few logging:
* when we are ingesting metrics and it takes more than 5 seconds, we log ERROR even though it's not an error. Metrics are still being ingested, they just take longer than 5 seconds. Change this logging to INFO
* added more logging when we return 500 to clients, we log which metrics failed

### CODE REVIEW

**Comment by ChandraAddala** [comment]
@shintasmith I think its 5 seconds, not 5 mins.

File: blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/processors/BatchWriter.java [hunk 1]
```diff
@@ -122,10 +122,13 @@ public Boolean call() throws Exception {
...
-                        if (System.currentTimeMillis() - writeStartTime > timeout.toMillis()) {
+                        long now = System.currentTimeMillis();
+                        if ( now - writeStartTime > timeout.toMillis()) {
                             exceededScribeProcessingTime.mark();
-                            getLogger().error("Exceeded timeout " + timeout.toString() + " before persisting " +
-                                    "all metrics for batch " + batchId);
+                            getLogger().info(
+                                    String.format("Batch write time %d (ms) exceeded timeout %s before persisting " +
+                                            "all metrics for batch %d",
+                                            now - writeStartTime, timeout.toString(), batchId));
```
Comment (ChandraAddala) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/processors/BatchWriter.java]: I agree that its not an error. But somebody thought its really bad that we are exceeding that timeout. Any idea what this timeout actually is?

Comment (shintasmith) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/processors/BatchWriter.java]: I have no idea... It seems arbitrary. It is hard coded inside HttpMetricsIngestionServer

Comment (ChandraAddala) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/processors/BatchWriter.java]: I did look into it a little bit. I vote for changing it to debug. There is meter associated with it. I plotted it and saved on ingest dashboard. Incase we wanted to find out if ingestion is taking more than 5 seconds, we can look at that dashboard.

http://graphite00.iad.prod.bf.k1k.me/grafana/#/dashboard/db/ingest-dashboard?panelId=31&fullscreen&edit

Comment (shintasmith) [inline blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/processors/BatchWriter.java]: I am ok with debug. I will change it.

**Comment by ChandraAddala** [comment]
Looks good.

**Comment by shintasmith** [comment]
@ChandraAddala , I changed the logging in BatchWriter to debug()

@usnavi , @VinnyQ , @izrik , pls review

**Comment by ChandraAddala** [comment]
Looks good.

**Pull request merged**